changelog/unreleased/3143

```diff
@@ -0,0 +1,8 @@
+Enhancement: Move to AndroidX Preference and new structure for settings
```

Comment:
I'd set is as `Feature`. Enhancement is like a scoped improvement in a feature. What do you think?

Reply:
I agree and though the same, but I wrote `Enhancement` as in the [README](https://github.com/owncloud/android/blob/feature/settings_androidx/changelog/README.md) it says that the four types of changes are: `Bugfix, Change, Enhancement, Security`. Enhancement is the one which fits the most from my point of view, let me know your opinion

Reply:
You're right, it fits the best.. `feature` is only for the branch name. My fault!